### PR TITLE
fby3: vf: Add plat_isr & plat_led &plat_util

### PR DESCRIPTION
**Pull request by HungYi-Li**
Summary:
- Add plat_isr
- Add plat_led
- Add plat_util

Test plan:
- Build code : This will be tested after a few commit.

Dependency: #446

### CODE REVIEW

File: meta-facebook/yv3-vf/src/platform/plat_hwmon.c [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	msg.rx_len = sizeof(buf);
+	memcpy(&msg.data[0], &reg, sizeof(reg));
+
+	if (i2c_master_read(&msg, retry)) {
+		printf("MB CPLD read failed!\n");
+		return false;
+	}
+
+	buf = msg.data[0];
+	buf = (buf & ~(1 << prsnt_ofs)) | (!val << prsnt_ofs);
```
Comment (garnermic) [inline meta-facebook/yv3-vf/src/platform/plat_hwmon.c:61]: Please resolve this linter issue on line 61.

File: meta-facebook/yv3-vf/src/platform/plat_led.c [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+		pre_state[idx] = ctrl;
+		SSDLEDSet(idx, ctrl);
+		break;
+	case SSD_START_BLINK:
+		k_timer_start(idx_to_ssdled_timer(idx), K_MSEC(500), K_MSEC(500));
+		break;
+	case SSD_STOP_BLINK:
+		stop_blink_timer(idx);
+		break;
+	}
```
Comment (garnermic) [inline meta-facebook/yv3-vf/src/platform/plat_led.c:101]: Please add a default case statement

File: meta-facebook/yv3-vf/src/platform/plat_power_seq.c [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+		clkbuf_oe_en(idx, en_clk);
+		delay_function(3, fm_p12v_sw_en, idx, en_12v);
+	}
+
+	if (force_ctl_3v3 && (enable == 0)) {
+		is_force_disable_3v3[idx] = 1;
+		fm_p3v3_sw_en(idx, en_3v3);
+	} else if (force_ctl_3v3 && (enable == 1)) {
+		is_force_disable_3v3[idx] = 0;
+	}
```
Comment (garnermic) [inline meta-facebook/yv3-vf/src/platform/plat_power_seq.c:200]: Please cover the else statement here.

File: meta-facebook/yv3-vf/src/platform/plat_power_seq.c [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+}
+
+uint8_t m2_dev_power_switch(uint8_t idx, uint8_t enable)
+{
+	return m2_dev_power_switch_with_pwrdis_chk(idx, enable, 1, 0);
+}
+
+uint8_t device_all_power_set(uint8_t idx, uint8_t set_val)
+{
+	uint8_t is_on = (set_val & DEV_PWR_ON ? 1 : 0);
```
Comment (garnermic) [inline meta-facebook/yv3-vf/src/platform/plat_power_seq.c:227]: Please fix the linter issues in lines 227, 228 and 229.

**Review by garnermic — CHANGES_REQUESTED** [review]
Few minor changes requested.

File: meta-facebook/yv3-vf/src/platform/plat_hwmon.c (hunk 1 above)
Comment (HungYi-Li) [reply meta-facebook/yv3-vf/src/platform/plat_hwmon.c:61]: reformat on #469

File: meta-facebook/yv3-vf/src/platform/plat_led.c (hunk 2 above)
Comment (HungYi-Li) [reply meta-facebook/yv3-vf/src/platform/plat_led.c:101]: add default on #469

File: meta-facebook/yv3-vf/src/platform/plat_power_seq.c (hunk 3 above)
Comment (HungYi-Li) [reply meta-facebook/yv3-vf/src/platform/plat_power_seq.c:200]: add else statement on #469

File: meta-facebook/yv3-vf/src/platform/plat_power_seq.c (hunk 4 above)
Comment (HungYi-Li) [reply meta-facebook/yv3-vf/src/platform/plat_power_seq.c:227]: fix it on #469

**Pull request closed without merging**